redbot/core/drivers/json.py

```diff
@@ -17,6 +17,7 @@
 _shared_datastore = {}
 _driver_counts = {}
 _finalizers = []
+_locks = {}
```

Comment:
Wouldn't it be simpler to use `defaultdict(asyncio.Lock)` and get rid of setter (and just use square bracket notation in property instead of `.get()`)?